pkg/storagecapabilities/storagecapabilities.go

```diff
@@ -172,6 +174,15 @@ var CloneStrategyByProvisionerKey = map[string]cdiv1.CDICloneStrategy{
 	"pd.csi.storage.gke.io/hyperdisk":          cdiv1.CloneStrategySnapshot,
 }
 
+// MinimumSupportedPVCSizeByProvisionerKey defines the minimum supported PVC size for a provisioner
+var MinimumSupportedPVCSizeByProvisionerKey = map[string]string{
+	"pd.csi.storage.gke.io/hyperdisk": "4Gi",
+	// https://aws.amazon.com/ebs/volume-types
+	"ebs.csi.aws.com/io1": "4Gi",
+	"ebs.csi.aws.com/io2": "4Gi",
+	"ebs.csi.aws.com/gp":  "1Gi",
```

Comment:
What about OCI? Is that a different PR?

Reply:
Citing @alromeros - OCI does not fail when requesting a small volume size, but provisions a 50Gi PVC.

Reply:
Exactly ^